package.json

```diff
@@ -14,7 +14,8 @@
     "lint": "yarn eslint \"./**/*.js\"",
     "ganache": "ganache-cli -p 8545 --gasLimit 0xfffffffffff --time '2017-05-10T00:00:00+00:00' -s 0",
     "publish": "lerna publish",
-    "test": "yarn clean && yarn compile && lerna run test --concurrency=1"
+    "test": "yarn clean && yarn compile && lerna run test --concurrency=1",
+    "check_dep_graph" : "source venv/bin/activate; python scripts/identify_dependencies.py"
```

Comment:
venv is in .gitignore so this command won't work unless people happen to have their venv setup the same way you do locally

Reply:
rather just remove source venv/bin/activate and make a note of usage in README.md or in the python_scripts directory.

Reply:
circleci may need to install python and virtualenv if this needs to be done.

Reply:
correct. working with link on that now.

Reply:
Please can you include a `requirements.txt` of the required venv setup to run the script? Such that after activating a venv locally you can just run `pip3 install -r requirements.txt` to get all requirements needed, and then run the script?

Reply:
I think the command is just `pip freeze > requirements.txt`

Reply:
extra space in `"check_dep_graph" : ` before the colon

Reply:
do you think that this command should now include `pip3 install -r requirements.txt` between the venv activate and running the script?